_includes/docs/latest/getting-started.html

```diff
@@ -2,7 +2,7 @@
   <h1 id="getting-started">Getting Started</h1>
 </div>
 
-<p class="lead">In version 2.5 we provide two modes: the stand alone "solo-server" mode and the heavier weight two server mode. The following describes the differences between the two modes.</p>
+<p class="lead">In version 3.0 we provide three modes: the stand alone "solo-server" mode, the heavier weight two server mode and distributed multiple-executor mode. The following describes the differences between the two modes.</p>
```

Comment:
As discussed offline, we are keeping all 3 modes for the time being as some user might want to use 2 serve mode.